test/libyul/evmCodeTransform/eof_swap17.yul

```diff
@@ -0,0 +1,228 @@
+object "main" {
+	code {
+	    function f(v00, v01, v02, v03, v04, v05, v06, v07, v08, v09, v10, v11, v12, v13, v14, v15, v16, v17) {
+            sstore(17, v17)
```

Comment:
`v17` is 17 slots deep in stack - all values in between are still needed - no way to avoid a `swapn{17}`